scripts/ci/lib.sh

```diff
@@ -154,7 +154,7 @@ get_central_debug_dump() {
 
     require_environment "API_ENDPOINT"
     require_environment "ROX_ADMIN_PASSWORD"
-    ROX_SERVER_NAME="" roxctl -e "${API_ENDPOINT}" \
```

Comment:
how this is related to the refactor? Does it fixes the issue? If so let's merge it separately from refactoring